src/Nethermind/Nethermind.Blockchain/FullPruning/PruningStatus.cs

```diff
@@ -31,4 +31,9 @@ public enum PruningStatus
     /// Full pruning was triggered and is starting.
     /// </summary>
     Starting,
+
+    /// <summary>
+    /// Pruning failed because of low disk space
+    /// </summary>
+    NotEnoughDiskSpace,
```

Comment:
very minor - thinking if we shoulnd't keep this enum in some order, so this would go after `Disabled`